diff --git a/vendor/cloudwatch-exporter/templates/_helpers.tpl b/helm/cloudwatch-exporter/templates/_helpers.tpl
index c4a3e7b..884bd3a 100644
--- a/vendor/cloudwatch-exporter/templates/_helpers.tpl
+++ b/helm/cloudwatch-exporter/templates/_helpers.tpl
@@ -35,6 +35,7 @@ Common labels
 */}}
 {{- define "yet-another-cloudwatch-exporter.labels" -}}
 helm.sh/chart: {{ include "yet-another-cloudwatch-exporter.chart" . }}
+application.giantswarm.io/team: {{ index .Chart.Annotations "io.giantswarm.application.team" | quote }}
 {{ include "yet-another-cloudwatch-exporter.selectorLabels" . }}
 {{- if .Chart.AppVersion }}
 app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
